src/map/items/item_furnishing.h

```diff
@@ -100,6 +100,67 @@ enum MOGHOUSE_AURA
     MOGLIFICATION_RESIST_CURSE         = 2855  // Slightly improves your resistance to curse effects
 };
 
+static const std::initializer_list<MOGHANCEMENT_TYPE> MoghancementList =
```

Comment:
i *really* don't like this - it's even only used in one single place (loading the moghancement). honestly, I think it'd just be better to store the current moghancement elsewhere in the db so we don't have to iterate all these keyitems on every load

Reply:
Did you have any places in mind? New column in chars table?

Reply:
chars_storage maybe? or chars, they both probably work fine

Reply:
as long as its not a char_vars I won't murder anybody :wink: